usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/ProcessParsedStopTime.java

```diff
@@ -86,7 +86,7 @@ public void execute(final ParsedEntity validatedParsedStopTime) {
         if (stopTime.isSuccess()) {
             if (gtfsDataRepository.addStopTime((StopTime) stopTime.getData()) == null) {
                 resultRepository.addNotice(new DuplicatedEntityNotice("stop_times.txt", null,
-                        validatedParsedStopTime.getEntityId(), "trip_id",
+                        "trip_id",
```

Comment:
Seems unrelated. Also, ecah parameter should be on their own line

Reply:
The constructor of `DuplicatedEntityNotice` that is used here has been modified, therefore this change might not seem related to the PR but is needed.

Reply:
Oh, alright. That make sense